Gemfile

```diff
@@ -3,4 +3,6 @@ source 'https://rubygems.org'
 spree_version = '~> 4.0'
 gem 'spree', spree_version
 
+gem 'addressable', '~> 2.8.6'
```

Comment:
I think this should be moved to gemspec..

I had commented this previously..seems like you missed this